csrc/python_frontend/translation.cpp

```diff
@@ -863,6 +864,59 @@ class FusionTranslator : public OptInConstDispatch {
         {fd_->recordingState(output())}));
   }
 
+  // Map PadOp to python frontend
+  void handle(const PadOp* pad_op) final {
+    Tensor output = fd_->defineTensor(pad_op->out()->as<TensorView>()->nDims());
+    map_val_to_fd_index_.emplace(pad_op->out(), output());
+
+    // Step 1: Get pad widths in normalized order.
+    std::vector<Val*> normalized_pad_widths = pad_op->getPadWidths();
+    const int64_t total_size = (int64_t)normalized_pad_widths.size();
+
+    // Step 2: Get indices for normalized pad widths.
+    std::vector<int64_t> normalized_indices(total_size);
+    std::iota(normalized_indices.begin(), normalized_indices.end(), 0);
+
+    // Step 3: Transform to indices for original pad widths
+    std::vector<int64_t> original_indices;
+    original_indices.reserve(normalized_indices.size());
+    std::transform(
+        normalized_indices.begin(),
+        normalized_indices.end(),
+        std::back_inserter(original_indices),
+        [=](int64_t normalized_idx) {
```

Comment:
This is just an for loop with irange?

Reply:
sorry, this just reads a bit strange to me. Didn't mean to ask you to change it.

Reply:
It is. Just having fun with `std::transform`